exchanges/binance/cfutures_types.go

```diff
@@ -204,21 +205,41 @@ type BatchCancelOrderData struct {
 	Msg           string  `json:"msg"`
 }
 
+// FuturesNewOrderRequest stores all the data needed to submit a
+// delivery/coin-margined-futures order.
+type FuturesNewOrderRequest struct {
+	Symbol           currency.Pair
+	Side             string
+	PositionSide     string
+	OrderType        string
+	TimeInForce      string
+	NewClientOrderID string
+	ClosePosition    string
+	WorkingType      string
+	NewOrderRespType string
+	Quantity         float64
+	Price            float64
+	StopPrice        float64
+	ActivationPrice  float64
+	CallbackRate     float64
+	ReduceOnly       bool
```

Comment:
Could you please add in field 'PriceProtect bool' and add in handing for that? Also in type `FuturesOrderPlaceData` could you please change `PriceProtect` to a `string` type? as it requires a "TRUE" or "FALSE" string from docs.